homeassistant/components/tibber/diagnostics.py

```diff
@@ -0,0 +1,39 @@
+"""Diagnostics support for Tibber."""
+from __future__ import annotations
+
+import tibber
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+REDACTED = "REDACTED"
```

Comment:
Perhaps wait for https://github.com/home-assistant/core/pull/64473?

Reply:
```suggestion
```